docs/cli-preview3/tools/dotnet-msbuild.md

```diff
@@ -0,0 +1,44 @@
+---
+title: dotnet-msmsbuild command | .NET Core SDK
+description: The dotnet-msmsbuild command provides access to MSmsbuild command line
+keywords: dotnet-msmsbuild, CLI, CLI command, .NET Core
+author: mairaw
+manager: wpickett
+ms.date: 10/13/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 70285a83-4103-4617-be8b-d0e1e9a4a91d
+---
+
+#dotnet-m/msbuild
+
+## Name 
+dotnet-msbuild -- msbuilds a project and all of its dependencies 

```

Comment:
This is the actual binary name...I'm not sure there should be a space there? Am I missing your point?